scss/styles.scss

```diff
@@ -2,5 +2,15 @@
   display: flex;
   flex-direction: column;
   height: 800px;
```

Comment:
I think there's no need for the height to be specified, just let the content decide the total height